.github/workflows/check_deps.yml

```
@@ -38,8 +38,33 @@ jobs:
       - name: Build
         run: make build
 
-      - name: Check dependencies
-        run: .github/workflows/check-deps.sh
-        env:
-          BRANCH_NAME: update-deps-${{ github.run_id }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update dependencies
+        id: update-deps
+        run: |
+          _update_output="$(TERM=dumb rebar3 update-deps --replace)"
+
+          cat <<EOF > /tmp/pr-body.md
+          Output of running \`rebar3 update-deps\`:
+
+          \```
+          $_update_output
+          \```
+          EOF
+
+          rebar3 fmt --write
+
+      # We can always run this step because the action will exit silently if there are no changes.
+      # See: https://github.com/marketplace/actions/create-pull-request#action-behaviour
+      - name: Create PR
+        uses: peter-evans/create-pull-request@v7
+        with:
+          # By always using the same branch name, we can keep pushing to
+          # the same branch if there are new changes.
+          branch: "automatic-dependencies-update"
+          commit-message: "Update dependencies"
+          title: "Update dependencies"
+          body-path: /tmp/pr-body.md
+          labels: "dependencies"
+          assignees: "weppos"
+          reviewers: "weppos"
```

Let's remove the assignees so whoever gets to review it self-assigns it.

Also,  let's make the platform team the default reviewer so everyone in the team gets noticed

```suggestion
          team-reviewers: "dnsimple/platform"
```

---

Yes, please 😄